index.js

```diff
@@ -27,6 +27,8 @@ app.post('/article/:uuid/:lang', (req,res) => {
 	const uuid = req.params.uuid;
 	const lang = req.params.lang;
 	const translators = req.body.translators;
+	// default is firstChunkOnly=true
+	const firstChunkOnly = (!req.query.hasOwnProperty('firstChunkOnly')) || Boolean(req.query.firstChunkOnly);
```

Comment:
Missing `new` in front of `Boolean`, another way to enforce Boolean value is to use a double bang `!!`